modules/monitoring/application/views/scripts/event/show.phtml

```diff
@@ -25,4 +25,5 @@ echo $object instanceof Service
     }
     ?>
     </table>
+    <?= $hookHtml ?>
```

Comment:
The host/service detailview extensions are rendered [before any standard html](https://github.com/Icinga/icingaweb2/blob/master/modules/monitoring/application/views/scripts/partials/object/detail-content.phtml#L4). I'd like to see this here as well. Does your use-case require this to be at the very end?

Reply:
No, our use case does not strictly require to be printed at the end. I will move it right before the "Event Details" table.